prysm/io.py

```diff
@@ -1375,3 +1375,94 @@ def read_sigfit_rigidbody(file):
             'dR': dR
         }
     return out
+
+def read_codev_int(filename):
+    """Read the content of a CodeV INT (.int/.cvg) file.
+    
+    Parameters
+    ----------
+    file : path_like
```

Comment:
the function arg is filename, but the docstring is file - please make them consistent